ui/page/components/treasury_list.go

```diff
@@ -0,0 +1,146 @@
+package components
+
+import (
+	"image/color"
+	"strings"
+
+	"gioui.org/layout"
+	"gioui.org/text"
+	"gioui.org/unit"
+	"gioui.org/widget"
+
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+type TreasuryItem struct {
+	Policy            dcrlibwallet.TreasuryKeyPolicy
+	OptionsRadioGroup *widget.Enum
+	VoteChoices       [3]string
+	SetChoiceButton   decredmaterial.Button
+}
+
+func (t *TreasuryItem) SetVoteChoices(voteChoices [3]string) {
+	t.VoteChoices = voteChoices
+}
+
+func TreasuryItemWidget(gtx C, l *load.Load, treasuryItem *TreasuryItem) D {
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			return layoutPiKey(gtx, l, treasuryItem.Policy)
+		}),
+		layout.Rigid(layoutVoteChoice(l, treasuryItem)),
+		layout.Rigid(func(gtx C) D {
+			return layoutPolicyVoteAction(gtx, l, treasuryItem)
+		}),
+	)
+}
+
+func layoutPiKey(gtx C, l *load.Load, treasuryKeyPolicy dcrlibwallet.TreasuryKeyPolicy) D {
+
+	var backgroundColor color.NRGBA
+
+	statusLabel := l.Theme.Label(values.TextSize14, treasuryKeyPolicy.PiKey)
+	backgroundColor = l.Theme.Color.LightBlue
+
+	return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			lbl := l.Theme.Label(values.TextSize20, values.String(values.StrPiKey))
+			lbl.Font.Weight = text.SemiBold
+			return layout.Flex{}.Layout(gtx,
+				layout.Rigid(lbl.Layout),
+			)
+		}),
+		layout.Rigid(func(gtx C) D {
+			return decredmaterial.LinearLayout{
+				Background: backgroundColor,
+				Width:      decredmaterial.WrapContent,
+				Height:     decredmaterial.WrapContent,
+				Direction:  layout.Center,
+				Alignment:  layout.Middle,
+				Border:     decredmaterial.Border{Color: backgroundColor, Width: values.MarginPadding1, Radius: decredmaterial.Radius(4)},
+				Padding:    layout.Inset{Top: values.MarginPadding3, Bottom: values.MarginPadding3, Left: values.MarginPadding8, Right: values.MarginPadding8},
```

Comment:
```suggestion
				Border:     decredmaterial.Border{
				Color: backgroundColor, 
				Width: values.MarginPadding1, 
				Radius: decredmaterial.Radius(4),
				},
				Padding:    layout.Inset{
				Top: values.MarginPadding3, 
				Bottom: values.MarginPadding3, 
				Left: values.MarginPadding8, 
				Right: values.MarginPadding8,
				},
```
This makes it easier to read.